assets/src/edit-story/app/font/fontProvider.js

```diff
@@ -78,45 +71,32 @@ function FontProvider({ children }) {
       };
 
       const defaultFontWeights = [{ name: fontWeightNames[400], value: 400 }];
-
       const currentFont = getFontByName(name);
-      let fontWeights = defaultFontWeights;
-      if (currentFont) {
-        const { weights } = currentFont;
-        if (weights) {
-          fontWeights = weights.map((weight) => ({
-            name: fontWeightNames[weight],
-            value: weight,
-          }));
-        }
+
+      if (!currentFont) {
+        return defaultFontWeights;
       }
-      return fontWeights;
-    },
-    [getFontByName]
-  );
 
-  const getFontFallback = useCallback(
-    (name) => {
-      const currentFont = getFontByName(name);
-      const fontFallback =
-        currentFont && currentFont.fallbacks ? currentFont.fallbacks : [];
-      return fontFallback;
+      const { weights } = currentFont;
+
+      return weights.map((weight) => ({
+        name: fontWeightNames[weight],
+        value: weight,
```

Comment:
Note: There are some string-number differences now, the weights dropdown is currently receiving a number for value, expecting string though.